### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by SpacePossum**
cc @GrahamCampbell

### CODE REVIEW

**Comment by julienfalque** [comment]
Do you mind handling https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3466#discussion_r165825898 here?

**Comment by SpacePossum** [comment]
that one should be done on 2.2, while this typo is on 2.10 and up only so we would still still need another PR for it I'm afraid

File: tests/IntegrationTest.php [hunk 1]
```diff
@@ -73,8 +73,8 @@ protected static function assertRevertedOrderFixing(IntegrationCase $case, $fixe
                 ], true)) {
                     static::markTestIncomplete(sprintf(
                         'Integration test `%s` was defined as explicit priority test, but no priority conflict was detected.'
-                        ."\n".'Either integration test needs to be extended, or test moved from `priority` to `misc`.'
-                        ."\n".'Bud don\'t do it blindly - it deserves investigation!',
+                        ."\n".'Either integration test needs to be extended or be moved from `priority` to `misc`.'
```
Comment (julienfalque) [inline tests/IntegrationTest.php]: Not sure about _"or be moved"_, maybe _"or moved"_ would be better?

**Comment by keradus** [comment]
oh boy... we need a spellchecker somehow :D

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request closed without merging**